pyproject.toml

```diff
@@ -41,17 +41,17 @@ dependencies = [
 # These optional dependencies are for building the
 # application as a part of a Github Action.
 ci = [
-    'pyinstaller==4.5.1',
+    'pyinstaller>=4.5.1',
     'pywin32-ctypes>=0.2.0; platform_system == "win32"', 
     'pypiwin32>=223; platform_system == "win32"',
     'pefile>=2022.5.30; platform_system == "win32"',
     'dephell_licenses>=0.1.7',
-    'scipy==1.6.1'
+    'scipy>=1.6.1'
 ]
```

Comment:
similarly to `pyinstaller`, `scipy` is known to break API often. Let's pin its version to whatever's current.

Reply:
Now pinned to minor version